web-client/public/js/sliders.js

```diff
@@ -133,14 +146,15 @@ export var sliderGroupController = function (sliderArray) {
             $("#link-distance").parent().removeClass("disabled");
             $("#charge").parent().removeClass("disabled");
         }
-
+*/
+// what does this function do?
         $.each(this.sliders, function (key, value) {
             $(value.sliderId).prop("disabled", !$(value.sliderId).prop("disabled"));
         });
```

Comment:
Do you mean the `each` call above? It toggles the `disabled` property of the elements with the given `sliderId`.